device.mk

```diff
@@ -205,4 +205,26 @@ PRODUCT_PROPERTY_OVERRIDES += \
 #
 PRODUCT_DEFAULT_PROPERTY_OVERRIDES += \
     persist.sys.usb.config=mtp \
```

Comment:
Yes, I agree with you and I also think we must fix this issue. But IMO a single broken feature should not this project. Before we get the manifest and all major repos landed, it is not easy for other module/function owners to join us doing verification of other developments.